jwst/lib/set_telescope_pointing.py

```diff
@@ -878,6 +878,13 @@ def update_wcs_from_telem(model, t_pars: TransformParameters):
         logger.warning('Calculation of S_REGION failed and will be skipped.')
         logger.warning('Exception is %s', e)
 
+    # If TARG_RA/TARG_DEC still 0/0 (e.g. pure parallels with no defined target),
```

Comment:
Suggestion: Add `logger.warning` when this occurs.

Reply:
Good call, this has been added.